library/src/main/java/com/yausername/youtubedl_android/YoutubeDL.kt

```diff
@@ -132,8 +133,18 @@ object YoutubeDL {
             }
         }
         return false
-    }
 
+    }
+    fun killChildProcesses() {
+        val process = Runtime.getRuntime().exec("ps -A -o pid,cmd")
+        process.inputStream.bufferedReader().useLines { lines ->
+            lines.filter { it.contains("ffmpeg") }
```

Comment:
Maybe aria2c too? Not sure about how many external executables in our libraies could have this issue

Reply:
See also: https://github.com/JunkFood02/Seal/issues/180

Reply:
And this function should actually be named `killAllFFmpegProcesses` according to its implementation